assets/section-main-product.css

```diff
@@ -1148,6 +1148,45 @@ a.product__text {
   border-color: rgb(var(--color-foreground));
 }
 
+.image-magnify-full-size {
+  cursor: zoom-out;
+  z-index: 1;
+  margin: 0;
```

Comment:
Here we're going to need to add some styling for the border radius I believe. I had not tested before with `global settings` and when [I go here to tweak the appearance](https://screenshot.click/20-59-lvnk9-q0xxm.mp4), it shows that the image is outside of it's containing box due to the border radius not being applied.

So that would need to be added.

Reply:
Great catch, thanks! I've added that in 👍